libpod/info.go

```diff
@@ -51,7 +52,29 @@ func (r *Runtime) hostInfo() (map[string]interface{}, error) {
 	if err != nil {
 		return nil, errors.Wrapf(err, "error parsing system uptime")
 	}
-	info["uptime"] = upDuration.String()
+
+	hoursFound := false
+	var timeBuffer bytes.Buffer
+	var hourBuffer bytes.Buffer
+	for _, elem := range upDuration.String() {
+		timeBuffer.WriteRune(elem)
+		if elem == 'h' || elem == 'm' {
+			timeBuffer.WriteRune(' ')
+			if elem == 'h' {
+				hoursFound = true
+			}
+		}
+		if !hoursFound {
+			hourBuffer.WriteRune(elem)
+		}
+	}
+
+	info["uptime"] = timeBuffer.String()
+	if hoursFound {
+		hours, _ := strconv.ParseFloat(hourBuffer.String(), 64)
```

Comment:
Can you do a `strings.Split("h")` here, and take the first element of the array to get only hours? That would guarantee you don't parse the rest

Reply:
@mheon I originally thought that doing the splitting might create a bunch of unnecessary strings and besides playing with a rune seemed like more fun!  I've just coded up the same functionality below using strings.Split().  I could probably tighten that up a bit more and make the naming better, but that's the gist of it.  

Let me know which approach you'd prefer and I'll finalize it and get a new PR pushed.

Reply:
My main goal is to get rid of the ignored error from ParseFloat() - to make sure we don't accidentally parse garbage and output the result. I suspected that ParseFloat() would error if it encountered non-float things, like a "h".
Given choice between the two, I think the first implementation is better, though i would like the unhandled error in ParseFloat handled. Sorry for the runaround on this one

Reply:
@mheon np, fun little experiment.  I'll work on handling the ParseFloat() error then.